backend/cmd/server.go

```diff
@@ -17,6 +18,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	if conf.ListPlugins {
+		if err := plugins.ListPlugins(conf.StaticDir, conf.PluginsDir); err != nil {
+			logger.Log(logger.LevelError, nil, err, "error listing plugins")
+			os.Exit(1)
+		}
+
+		os.Exit(0)
```

Comment:
Why are we exiting here? This can be removed otherwise server will close.

Reply:
That is the idea, to have the server terminate after running the command. It's simply to list plugins without running the server